README.md

```diff
@@ -64,40 +82,41 @@ the pool and error is logged.
 
 Configuration is read from the yaml configuration file with the following fields:
```

Comment:
A bit inconsistent: in the example, the configuration is a JSON file, here we get YAML. Maybe we could find some information on supported config formats here then? Anyway "from a yaml configuration file" (first mention)